nixos/tests/auto-cpufreq.nix

```diff
@@ -0,0 +1,33 @@
+import ./make-test-python.nix (
+  { pkgs, ... }:
+
+  {
+    name = "auto-cpufreq-server";
+
+    nodes = {
+      machine =
+        { pkgs, ... }:
+        {
+          # service will still start but since vm inside qemu cpufreq adjustments
+          # cannot be made. This will resource in the following error but the service
+          # remains up:
+          #   ERROR:
```

Comment:
Isn't this error misleading? I guess it's documented here and thus expected, but still not great.

Reply:
I agree but theres no way Im aware of to supress the error inside of qemu